controllers/mattermost/mattermost/mattermost_test.go

```diff
@@ -547,3 +547,59 @@ func TestCheckMattermostExternalDBAndFileStore(t *testing.T) {
 		})
 	})
 }
+
+func TestSpecialCases(t *testing.T) {
+	// Setup logging for the reconciler so we can see what happened on failure.
+	logger := blubr.InitLogger()
+	logger = logger.WithName("test.opr")
+	logf.SetLogger(logger)
```

Comment:
Looks like the tabbing is a little off here maybe?

Reply:
I think it is fine as `go fmt` is passing without any changes. It might just look a little weird here on GitHub.